.vscode/settings.json

```diff
@@ -12,5 +12,7 @@
   "[markdown]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "prettier.configPath": "\"C:\\AVMMain\\bicep-registry-modules\\.editorconfig\""
```

Comment:
Please undo. Those settings seem to be very much local ;)

Reply:
Done

Reply:
You may have missed to push your changes @jeetgarg  ;)